assets/js/modules/thank-with-google/datastore/publications.test.js

```diff
@@ -348,5 +352,67 @@ describe( 'modules/thank-with-google publications', () => {
 				);
 			} );
 		} );
+
+		describe( 'getServiceCreatePublicationURL', () => {
+			it( 'returns an account chooser URL with the home URL appended to the publisher center URL', () => {
+				provideSiteInfo( registry );
+				provideUserInfo( registry );
+
+				const homeURL = registry.select( CORE_SITE ).getHomeURL();
+				const publisherCenterURL = new URL(
+					'https://publishercenter.google.com/'
+				);
+				publisherCenterURL.searchParams.set(
+					'sk_url',
+					encodeURIComponent( homeURL )
+				);
```

Comment:
Same here, let's stick with `addQueryArgs`.

Reply:
Addressed this as well, thank you!